sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -891,123 +874,48 @@ describe("EventHub Receiver", function(): void {
   });
 
   describe("Negative scenarios", function(): void {
-    describe("on invalid partition ids like", function(): void {
-      const invalidIds = ["XYZ", "-1", "1000", "-"];
-      invalidIds.forEach(function(id: string): void {
-        it(`"${id}" should throw an error`, async function(): Promise<void> {
-          try {
-            debug("Created receiver and will be receiving messages from partition id ...", id);
-            const d = await client
-              .createConsumer(EventHubConsumerClient.defaultConsumerGroupName, id, latestEventPosition)
-              .receiveBatch(10, 3);
-            debug("received messages ", d.length);
-            throw new Error("Test failure");
-          } catch (err) {
-            debug("Receiver received an error", err);
-            should.exist(err);
-            err.message.should.match(
-              /.*The specified partition is invalid for an EventHub partition sender or receiver.*/gi
-            );
-          }
-        });
+    it("should throw MessagingEntityNotFoundError for non existing consumer group", async function(): Promise<
+      void
+    > {
+      const badConsumerClient = new EventHubConsumerClient(
+        "boo",
```

Comment:
This is how I know you're the test author ;)

Reply:
I don't know why one would choose "foo" and "bar" when you have "boo" and "hoo" :)